k8s/perftest/common/monitoring/oneagent-values.yaml

```diff
@@ -6,9 +6,10 @@ metadata:
   namespace: monitoring
 spec:
   encryptedData:
-    values.yaml: AgBwFV4H3ECXCd6v8ArHeEtVRLCEpOw60qcHIVTSGZrR8+Zq6XJt+SZxYusBRcBOHY5qcMznGUDp+J6Z8qHoNhD5gOXCbhPgfi9otLCwJk2h/AE9Gk0047/795ENFIu4DuA89TB9GHiS9smjX6qe0feVEM1tutrAWBFVSeYIEeMAPBm7v1OgKE3NuX21a/lFISAYD7IW+Y/2njCwdp0bt4xn9QSaTkcqDo3eDGK6u3zgxVBaE8XXPN3MKkUjRaNlKZue29y6gjwlYqR6ymxYMrro1tJRNuLoKQdhSx5t+lIllVWYf8o609OVXqHIeWLgdl3jBYxW0Jlt+RDcLO3H0GWHs8aPG+YzMfpo0vvpWpdZjbQaIUBaUW3phnb2FQSIZaj2LsiqzYxBcSWUJu1ZsTsfF41xiGAPSGUV51vp4jpy4WTg2WTHmmIzzod1igV3IYO2t326AXl3Zho86BupOki0dp+O7TFTa4bLnN4ZHfPN65icmkp1R5I5AuF316xO7aqWlIKVsgQ3LyKQc3lgv0XhSkDr7zfRax6G6cvKeK3kU3gSD4sOeNNkeh2tigkS//td9Fzzapy5aJ8dXWEnQC1iT9emrxXS/hHJo8E55JiF9CLWylFzZgLa9rJ6ebUqCp9JarkbeJJxGSHC8C6loxVsXQW3h7P6IuTq1ObDx3h6lV7xCBUbhzS2TnpqwFttgCABgMl2IPCc/zqw6gu2xj9N+7u30BwGRjFD0zRpUWeWFLVpJTgGlc4q8JIxdHjboqZjDwoz+1qt6zew3MgHb1hAREWJZR1Gb6r2oObmeQ==
+    key: AgAzXfSTGsQzjPNuyihtdnakG+R8+Fn3l7fEonZ9ykwoR7g+AK/W1s2suJFD6Q+sWYvZrQf5jGQtu8nAgKQ7L1yq3LgDJE/CuBdv03f4dKN/K4U50c5SePK1yYmhmXBjpM64V+p5a+iG4xV6si0ZMGlWKInkJ19LJHIgXZAFhLWvCIlB1iJAWX92VgfImXfsQTA8+sHe7zu5UfijWruQ1G2StwytOmE2kIxJ1Y0KniARhukQvf88YyUT/WgExWNDBVH5sWOYwHD0Mfcv59FbKsXjjIbrdo6pLVfRA2HOVsmMOSOFRs26cXrUelBBSHusJV5R8gyCByS6H3SWyTLe2mXyyFc2OFQGk2BzhoA28hCVU+Qa3SZV0ykrgIlGrWw3M2NUIJ4iDuzYIG4gy8WrmZAYqgkbY0QeCJnczZjuv/6IOUTKTwlyvTbWRN1TWV/RKLwx8xiDjAsKpH1siFEbnE/SD9QH/3Eyhv3GmlIxFa8K+HuHjkpsfD90/KmFpBVFGlumXEGFWHD/M/VcAfUaTInJpRwbnZO/X0crhaXQMydtp+36SI7Gptc+bB11yMXeVWB9HK6x5RJedXXAB9MMahc/+Gc+x/JCHo0JQoivOH1/OvnwUB4o4ccmGhJ/GNW3m8CbYeFAgkKJ8rngsAfutOAyTNhl7wnyzT2hFx4SmTeH25rh1/gU1ZbHGKwf9GZ9KVwrgmhxyD8D6dA3+3rTLQqXCIZeg1c=
```

Comment:
Shouldn’t this be values.yaml?

Reply:
that's what the kubeseal generated for me. The version of kubeseal I am using is 0.8.1. Not sure if I should change that manually?

Reply:
Probably not sealed properly.